src/signals/incident-management/containers/IncidentDetail/components/MetaList/MetaList.test.tsx

```diff
@@ -23,14 +23,14 @@ import { fetchCategoriesSuccess } from 'models/categories/actions'
 import * as departmentsSelectors from 'models/departments/selectors'
 import * as categoriesSelectors from 'models/categories/selectors'
 
-import IncidentDetailContext from '../../../context'
+import IncidentDetailContext from '../../context'
 import {
   mockRequestHandler,
   fetchMock,
-} from '../../../../../../../../internals/testing/msw-server'
-import * as API from '../../../../../../../../internals/testing/api'
-import MetaList from '../MetaList'
-import type { IncidentChild } from '../../../types'
+} from '../../../../../../../internals/testing/msw-server'
+import * as API from '../../../../../../../internals/testing/api'
```

Comment:
we zouden voor dit soort imports ook een alias kunnen aanmaken in webpack.base.babel.js'

ala
    alias: {
      types: path.resolve(__rootdir, 'src/types/'),
    },

Reply:
ik heb van alles geprobeerd maar krijg het niet voor elkaar. Ik denk omdat `internals` buiten `src` ligt.